docs/plugin-development/restrictions.md

```diff
@@ -82,6 +82,67 @@ start debugging performance issues is through the Plugin Statistics window,
 which can be found through Plugins > Open Plugin Stats in the dev menu
 (`/xldev`).
 
+## Can my plugin talk to a backend server I run?
+
+Plugins are permitted to communicate with special backend servers, though there
+are certain considerations and requirements that must be met:
+
+- Plugins should take care to send the minimum amount of data necessary to do
+  their job. Whenever feasible, plugins should hash information like Content IDs
+  or player names on the client side so that a server-side data breach does not
```

Comment:
I believe this is targeted towards the local player name, not names in general? The idea being not to share identifiable information of the user if it's not needed.

Reply:
Edited for clarity.